documentation/src/main/asciidoc/reference/mapper-orm-indexing-massindexer.asciidoc

```diff
@@ -321,7 +321,14 @@ the different options applied to the `MassIndexer` affect
 the number of used worker threads and connections:
 
 ----
-threads = typesToIndexInParallel * (threadsToLoadObjects + 1);
+if ( using the default 'none' coordination strategy ) {
+    threadsToCoordinate = 0;
+}
+else {
+    threadsToCoordinate = 1;
+}
+threadsToLoadIdentifiers = 1;
+threads = threadsToCoordinatetypesToIndexInParallel * (threadsToLoadObjects + threadsToLoadIdentifiers);
```

Comment:
I cannot see where `threadsToCoordinate` variable is used in this example.

Reply:
There was a typo: `threadsToCoordinatetypesToIndexInParallel * ...` should have  been `threadsToCoordinate + typesToIndexInParallel * ...`